### DISCUSSION TITLE
Netlog password

### DISCUSSION THREAD

**Discussion by Calamitou** [3kh0/ext-remover#398] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
i found my school wifi passcode using this but i cant unblock anything

**Comment by Aka-but-nice** [comment, marked as answer]
BRO you got it, now you can change your DNS to a custom one, so extensions stop working

**Reply by Calamitou** [reply]
cool but idk how to do that and i dont wanna be suspended for it, and i also dont wanna downgrade like my friend who did it without being caught

**Reply by Aka-but-nice** [reply]
They won't ever realize, it's because schools would have to directly look at your screen to see what you're on, but since you are not on their DNS they can't check, and also since there are so many people in your school, I'm sure they won't notice one person.

**Reply by Calamitou** [reply]
then how do i do it?

**Reply by ennzo1010** [reply]
> BRO you got it, now you can change your DNS to a custom one, so extensions stop working

How do you change your DNS to a custom one so the extensions stop working? Btw i have my school wifi pass.

**Reply by Calamitou** [reply]
idk, ask him

**Comment by 3kh0** [comment]
Wow you got it with this tool: https://luphoria.com/netlog-policy-password-tool

**Reply by Calamitou** [reply]
so should I use it?

**Reply by 3kh0** [reply]
yes

**Reply by Calamitou** [reply]
lol says its under illegal activity

**Comment by Alex2Mayhem** [comment]
> i found my school wifi passcode using this but i cant unblock anything

Whats the passcode?

**Reply by Calamitou** [reply]
... why should i tell you

**Reply by Aka-but-nice** [reply]
It wouldn't work, because there's a 0.1 chance that y'all are in the same school!

**Reply by Alex2Mayhem** [reply]
i know it wouldn't work.

**Reply by Aka-but-nice** [reply]
Do you mean how to get the passcode? Because I can tell you, just go to <a href="hhttps://nppe.glitch.me/">here</a> and follow the steps there.

**Reply by rwg100** [reply]
> ... why should i tell you

nothing will happen you know? besides getting FREE WIFI EVERYWHERE YOU GO!!!!!!!!!!!!!11!!!!

**Reply by Aka-but-nice** [reply]
That's not how wifi works, you would have to be in the specific location the wifi modem is (e.g., school, home, or place.)

**Reply by Calamitou** [reply]
well if i tell you u can decode it and know my wifi name and where i live
